binder/Dockerfile

```diff
@@ -1 +1 @@
-FROM ghcr.io/openradar/erad2024:d6ee5b14f782
+FROM ghcr.io/openradar/erad2024:pr_68
```

Comment:
Same here. This will make sure, that the image which is build on push to main is used in subsequent pythia-binder instances.